pkg/util/timeutil/time_zone_util.go

```diff
@@ -66,3 +67,45 @@ func ParseFixedOffsetTimeZone(location string) (offset int, origRepr string, suc
 	}
 	return offset, strings.TrimSuffix(strings.TrimPrefix(origRepr, "("), ")"), true
 }
+
+// TimeZoneOffsetStringConversion converts a time string to offset seconds
+// Supported time zone strings :- GMT/UTC±[00:00:00 - 15:59:59], GMT/UTC±[0-16]
+// Unsupported time zone strings :- GMT/UTC±16:00 to upper, GMT/UTC±6.5
```

Comment:
Sorry for thinking 15:59 was the min/max, but turns out postgres allows up to `167:59:00`...

```
otan=# set time zone 'UTC+168';
ERROR:  invalid value for parameter "TimeZone": "UTC+168"
otan=# set time zone 'UTC+167:59';
SET
otan=# set time zone 'UTC+167:59:59';
ERROR:  time zone "UTC+167:59:59" appears to use leap seconds
DETAIL:  PostgreSQL does not support leap seconds.
otan=# set time zone 'UTC+167:59:00';
SET
otan=# set time zone 'UTC+167:59:00';
SET
otan=# set time zone 'UTC+167:59:01';
ERROR:  time zone "UTC+167:59:01" appears to use leap seconds
DETAIL:  PostgreSQL does not support leap seconds.
otan=# set time zone 'UTC-167:59:00';
SET
otan=# set time zone 'UTC-168';
ERROR:  invalid value for parameter "TimeZone": "UTC-168"
otan=# set time zone 'UTC+168';
ERROR:  invalid value for parameter "TimeZone": "UTC+168"
```

Reply:
Good to know. This will be resolved once the above changes are made :)